provider/clusterrosaclassic/cluster_rosa_classic_datasource.go

```diff
@@ -0,0 +1,399 @@
+				Computed:    true,
+			},
+			"private": schema.BoolAttribute{
+				Description: "Restrict cluster API endpoint and application routes to, private connectivity. This requires that PrivateLink be enabled and by extension, your own VPC. " + common.ValueCannotBeChangedStringDescription,
+				Computed:    true,
+			},
+			"availability_zones": schema.ListAttribute{
+				Description: "Availability zones. " + DefaultMachinePoolMessage,
+				ElementType: types.StringType,
+				Computed:    true,
+			},
+			"machine_cidr": schema.StringAttribute{
+				Description: "Block of IP addresses for nodes. " + common.ValueCannotBeChangedStringDescription,
+				Computed:    true,
+			},
+			"proxy": schema.SingleNestedAttribute{
+				Description: "proxy",
+				Attributes:  proxy.ProxyDatasource(),
+				Computed:    true,
+			},
+			"service_cidr": schema.StringAttribute{
+				Description: "Block of IP addresses for the cluster service network. " + common.ValueCannotBeChangedStringDescription,
+				Computed:    true,
+			},
+			"pod_cidr": schema.StringAttribute{
+				Description: "Block of IP addresses for pods. " + common.ValueCannotBeChangedStringDescription,
+				Computed:    true,
+			},
+			"host_prefix": schema.Int64Attribute{
+				Description: "Length of the prefix of the subnet assigned to each node. " + common.ValueCannotBeChangedStringDescription,
+				Computed:    true,
+			},
+			"current_version": schema.StringAttribute{
+				Description: "The currently running version of OpenShift on the cluster, for example '4.11.0'.",
+				Computed:    true,
+			},
+			"state": schema.StringAttribute{
+				Description: "State of the cluster.",
+				Computed:    true,
+			},
+			"ec2_metadata_http_tokens": schema.StringAttribute{
+				Description: "This value determines which EC2 Instance Metadata Service mode to use for EC2 instances in the cluster." +
+					"This can be set as `optional` (IMDS v1 or v2) or `required` (IMDSv2 only). This feature is available from " +
+					"OpenShift version 4.11.0 and newer. " + common.ValueCannotBeChangedStringDescription,
+				Computed: true,
+			},
+			"private_hosted_zone": schema.SingleNestedAttribute{
+				Description: "Used in a shared VPC topology. HostedZone attributes. " + common.ValueCannotBeChangedStringDescription,
+				Attributes: map[string]schema.Attribute{
+					"id": schema.StringAttribute{
+						Description: "ID assigned by AWS to private Route 53 hosted zone associated with intended shared VPC, " +
+							"e.g. 'Z05646003S02O1ENCDCSN'.",
+						Computed: true,
+					},
+					"role_arn": schema.StringAttribute{
+						Description: "AWS IAM role ARN with a policy attached, granting permissions necessary to " +
+							"create and manage Route 53 DNS records in private Route 53 hosted zone associated with " +
+							"intended shared VPC.",
+						Computed: true,
+					},
+				},
+				Computed: true,
+			},
+
+			// Deprecated Attributes:
```

Comment:
Please add here more informative comment:
* What are the deprecated attributes?
* Why do you mark them
* Why these attributes can't just be removed?

Second, is it possible to mark these attributes as deprecated to avoid generating docs?
I don't remember if we had discussion about this
But maybe something can be done here, like this:
https://developer.hashicorp.com/terraform/plugin/sdkv2/best-practices/deprecations#provider-attribute-removal

Reply:
I added a comment
According to your second point - I tried to find in the tfplugindoc but there is no way to remove attributes from the documentation. 
It looks like: 
` `disable_waiting_in_destroy` (Boolean, Deprecated) Disable addressing cluster state in the destroy resource. Default value is false, and so a `destroy` will wait for the cluster to be deleted.`